src/app/onion/learning-object-builder/components/editor-action-panel/edit-changelog/edit-changelog.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="changelog">
-  <clark-text-editor editorPlaceholder="What editorial changes were made to this learning object?" [(savedContent)]="changelog" (textOutput)="changelogChange.emit($event)" name="changelog"></clark-text-editor>
+  <clark-text-editor editorPlaceholder="What {{ mode }} changes were made to this learning object?" [(savedContent)]="changelog" (textOutput)="changelogChange.emit($event)" name="changelog"></clark-text-editor>
```

Comment:
I think we might be over complicating this placeholder text. The sentence `What author changes were made to this learning object?` doesn't seem to provide any value over `What changes were made to this learning object?`

I think we should keep it simple and avoid adding another input property.